gitcoinco/web
# Hall of fame

##### Description




- A model was created for holding the dynamic data of the "Hall of fame" page.
- The GrantHallOfFame object that shall be used in the page needs to be published. This can be done via an admin action in the admin list page. Only 1 object can have the published status set to 'true'.

Open point:
- have left the background of the "share your story section" white, as the picture is not transparent


##### Refers/Fixes

#9413

##### Testing

No automatic test cases are provided.

Prerequisites:
1. Open http://localhost:8000/_administrationgrants/granthalloffame/
2. Create a hall of fame object
3. Publish that object (with the 'Publish' action from the admin list view)

Ways to navigate to the page:
1. Open the "Hall of Fame" page via direct link: http://127.0.0.1:8000/grants/hall_of_fame (when run locally)
2. Navigate to the "Hall of Fame" page via top menu: Products -> Grants -> Hall of Fame
3. Open the grants page. Navigate to the "Hall of Fame" page via the link "see hall of fame" below the "Create a Grant" button
